sdk/confidentialledger/azure-security-confidentialledger/src/test/java/com/azure/security/confidentialledger/ConfidentialLedgerIdentityClientTestBase.java

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+
+package com.azure.security.confidentialledger;
+
+import com.azure.core.test.TestBase;
+
+class ConfidentialLedgerIdentityClientTestBase extends TestBase {
```

Comment:
what is this for?

Reply:
This was autogenerated :) Should I remove it?